localfs_windows.go

```diff
@@ -0,0 +1,125 @@
+// +build windows
```

Comment:
- Since the filename is already suffixed with `_windows` the extra build tag isn't actually needed here.

- I believe localfs.go needs build tags to exclude windows, or the build will fail with name conflicts on Windows.